test/autoprefixer.test.js

```diff
@@ -547,6 +547,13 @@ it('works with CSS Modules', () => {
   postcss([autoprefixer()]).process(':export { selectors: _1q6ho_2 }').css
 })
 
+it('ignore prefix in vendor at rules', () => {
+  let input = read('at-rules')
+  let output = read('at-rules.out')
+  let result = postcss([autoprefixer({ cascade: false })]).process(input)
```

Comment:
`autoprefixer` will use current browsers and could change output in the future

Reply:
I fixed that.